### ISSUE TITLE
incompatibility with (neo)forge =/= 47.1.3

### ISSUE THREAD

**Issue by cfnnit** [Skidamek/AutoModpack#183]
### Bug description

The server is not running.

### Steps to reproduce

1. Added automodpack to the server.
2. The server is not running.

### Relevant logs

```
[init] Using Aikar's >12GB flags
[init] Setting initial memory to 16G and max to 16G
[init] Using Forge supplied run.sh script...
WARNING: Using incubator modules: jdk.incubator.vector
2023-10-19 21:48:43,513 main WARN Advanced terminal features are not available in this environment
[21:48:43] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher running: args [--launchTarget, forgeserver, --fml.forgeVersion, 47.2.1, --fml.mcVersion, 1.20.1, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20230612.114412]
[21:48:43] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher 10.0.9+10.0.9+main.dcd20f30 starting: java version 17.0.8 by Oracle Corporation; OS Linux arch amd64 version 6.1.0-13-amd64
[21:48:44] [main/INFO] [ne.mi.fm.lo.ImmediateWindowHandler/]: ImmediateWindowProvider not loading because launch target is forgeserver
[21:48:44] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/data/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%2365!/ Service=ModLauncher Env=SERVER
[21:48:44] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/fmlcore/1.20.1-47.2.1/fmlcore-1.20.1-47.2.1.jar is missing mods.toml file
[21:48:44] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/javafmllanguage/1.20.1-47.2.1/javafmllanguage-1.20.1-47.2.1.jar is missing mods.toml file
[21:48:44] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/lowcodelanguage/1.20.1-47.2.1/lowcodelanguage-1.20.1-47.2.1.jar is missing mods.toml file
[21:48:44] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/mclanguage/1.20.1-47.2.1/mclanguage-1.20.1-47.2.1.jar is missing mods.toml file
[21:48:44] [main/WARN] [ne.mi.ja.se.JarSelector/]: Attempted to select two dependency jars from JarJar which have the same identification: Mod File:  and Mod File: . Using Mod File: 
[21:48:44] [main/WARN] [ne.mi.ja.se.JarSelector/]: Attempted to select a dependency jar for JarJar which was passed in as source: curios. Using Mod File: /data/mods/curios-forge-5.3.5+1.20.1.jar
[21:48:44] [main/WARN] [ne.mi.ja.se.JarSelector/]: Attempted to select a dependency jar for JarJar which was passed in as source: resourcefullib. Using Mod File: /data/mods/resourcefullib-forge-1.20.1-2.1.11.jar
[21:48:44] [main/INFO] [ne.mi.fm.lo.mo.JarInJarDependencyLocator/]: Found 28 dependencies adding them to mods collection
[21:48:44] [main/INFO] [PreloadingTricks/LanguageProvider/]: [PreloadingTricks/LanguageProvider] Loading settingdust.preloadingtricks.fabric.FabricLanguageProviderCallback
[21:48:44] [main/INFO] [PreloadingTricks/LanguageProvider/]: [PreloadingTricks/LanguageProvider] Loading settingdust.preloadingtricks.quilt.QuiltLanguageProviderCallback
[21:48:44] [main/INFO] [PreloadingTricks/LanguageProvider/]: [PreloadingTricks/LanguageProvider] Loading settingdust.preloadingtricks.forge.ForgeLanguageProviderCallback
[21:48:44] [main/INFO] [PreloadingTricks/ModSetup/]: [PreloadingTricks/ModSetup] Loading pl.skidam.automodpack_core.Preload[21:48:44] [main/INFO] [AutoModpack/]: Prelaunching AutoModpack...
[21:48:44] [main/INFO] [AutoModpack/]: Loaded config! took 4ms
[21:48:44] [main/INFO] [AutoModpack/]: Checking if AutoModpack is up-to-date...
[21:48:45] [main/INFO] [AutoModpack/]: Didn't find any updates for AutoModpack! You are on the latest version: 3.5.2
[21:48:45] [main/INFO] [AutoModpack/]: AutoModpack prelaunched! took 430ms
[21:48:45] [main/ERROR] [ne.mi.fm.lo.ModSorter/LOADING]: Missing or unsupported mandatory dependencies:
	Mod ID: 'forge', Requested by: 'fabric_networking_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
	Mod ID: 'forge', Requested by: 'fabric_registry_sync_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
	Mod ID: 'forge', Requested by: 'fabric_command_api_v2', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
	Mod ID: 'forge', Requested by: 'fabric_api_base', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
	Mod ID: 'forge', Requested by: 'fabric_resource_loader_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
Exception in thread "main" java.lang.module.ResolutionException: Modules net.fabricmc.loader and creativecore export package net.fabricmc.api to module lolmha
	at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
	at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
	at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
	at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
	at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
	at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:75)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
2023-10-19T21:48:47.702+0900	WARN	mc-server-runner	Minecraft server failed. Inspect logs above for errors that indicate cause. DO NOT report this line as an error.	{"exitCode": 1}
```

### Minecraft & Mod Loader versions

1.20.1 forge 47.2.1

### Operating system

Linux

### AutoModpack version

3.5.2

### Other information

https://github.com/Skidamek/AutoModpack/issues/176

This was a similar issue, but it quickly closed itself.

### Check list

- [X] I have verified that the issue persists in the latest version of the mod.
- [X] I have searched the existing issues and confirmed that this is not a duplicate.

**Comment by haruww** [comment]
try using 3.5.3

**Comment by darkphan** [comment]
> try using 3.5.3

From where?

**Comment by darkphan** [comment]
I found the build for 3.5.3 and it does not work with forge versions above 47.1.3

**Comment by haruww** [comment]
it worked with neoforge

**Comment by darkphan** [comment]
This happens with Forge and Neoforge. 

![FN72adh](https://github.com/Skidamek/AutoModpack/assets/2169426/9728e4c4-58e9-450a-a167-f651e687a5dd)

**Comment by Skidamek** [comment]
Install last version of this [mod](https://modrinth.com/mod/forgified-fabric-api) along side the automodpack and it should launch

**Comment by Skidamek** [comment]
Fixed in 4.0.0 beta 1!
